s2.asm

```diff
@@ -798,9 +798,9 @@ SS_PNTA_Transfer_Table:	offsetTable
 	startZ80
 
 	bsr.w	ProcessDPLC2
-	tst.w	(Demo_Time_left).w
-	beq.w	+	; rts
-	subq.w	#1,(Demo_Time_left).w
+	tst.w	(Demo_Time_left).w	; is there time left on the demo?
+	beq.w	+	; if not, return
+	subq.w	#1,(Demo_Time_left).w	; subtract 1 from time left in demo
```

Comment:
shouldn't comments that are basically;
`x = x + 1; // add 1 to x`
be omitted?

Reply:
If my memory's right, the other instances of this code already had comments like these, so it's more of a consistency thing.

Granted, I would prefer this disassembly have little of its fat trimmed, but that is a lot of code to sift through. You also have the fact that coding novices read through this.